src/store/mutations.js

```diff
@@ -182,10 +182,24 @@ function resetRequests(state) {
 }
 
 function setAcceptCommitInfo(state, value) {
+  if (value === false && storage.get('commitInfoAccepted')) {
+    storage.delete('commitInfoAccepted');
+  }
+
+  if (value === true) {
+    storage.set('commitInfoAccepted', true);
+  }
   state.acceptCommitInfo = value;
 }
 
 function setAcceptDexDemoVersion(state, value) {
+  if (value === false && storage.get('dexInfoAccepted')) {
```

Comment:
It doesn’t need to read the storage here it can use `state.acceptDexDemoVersion` instead of doing `storage.get`

Reply:
Does it need it at all in that case? I was just making sure that it was in localStorage before attempting to delete it. storage.delete may actually take care of that and then it can be just `if(value === false) delete` ?

Reply:
I suggest just using something like this should be safe:
```
if (!value && state.acceptDexDemoVersion) {
  storage.delete(‘dexInfoAccepted’);
} else if (value && !state.acceptDexDemoVersion) {
  storage.set(‘dexInfoAccepted’, true);
}
```

Reply:
Did need the second level array check of currentNetwork ([commit](https://github.com/aphtoken/aph-desktop-web-dex/pull/6/commits/68a292f86cf7f5d3f2b2e99b5f709c9cc6bfbb7b)); tested and works well. Also made this change for the commitInfo too. Should be good to go?

Reply:
I guess it is ok that the saved state from one network will propagate to the other network while logged in, since saved state for these is not kept per network.